src/components/test-slot/test-outcome/test-outcome.ts

```diff
@@ -190,13 +192,15 @@ export class TestOutcomeComponent implements OnInit {
     }
   }
 
-  startTest() {
+  async startTest() {
     if (this.isE2EPracticeMode()) {
       // this.store$.dispatch(StartE2EPracticeTest(this.slotDetail.slotId.toString()));
     } else {
       this.store$.dispatch(StartTest(this.slotDetail.slotId, this.category, this.startTestAsRekey || this.isRekey));
     }
-    // this.router.navigate(this.getTestStartingPage());
+    // TODO - temp to allow test to start without data
+    this.category = TestCategory.B;
```

Comment:
are we able tore move this now, and should we be using getTestStartingPage() underneath instead of hardcoding WAITING_ROOM_PAGE

Reply:
yes we are. and getTestStartingPage is now redundant. but i will look to replace the string with an enum